pkg/blob/blob.go

```diff
@@ -266,20 +266,25 @@ func (d *Driver) Run(endpoint, kubeconfig string, testBool bool) {
 }
 
 // GetContainerInfo get container info according to volume id, e.g.
-// input: "rg#f5713de20cde511e8ba4900#containerName#uuid"
-// output: rg, f5713de20cde511e8ba4900, containerName
-// input: "rg#f5713de20cde511e8ba4900#containerName#uuid#namespace"
-// output: rg, f5713de20cde511e8ba4900, containerName, namespace
-func GetContainerInfo(id string) (string, string, string, string, error) {
+// input: "rg#f5713de20cde511e8ba4900#containerName#uuid#"
+// output: rg, f5713de20cde511e8ba4900, containerName, "" , ""
+// input: "rg#f5713de20cde511e8ba4900#containerName#uuid#namespace#"
+// output: rg, f5713de20cde511e8ba4900, containerName, namespace, ""
+// input: "rg#f5713de20cde511e8ba4900#containerName#uuid#namespace#subsID"
+// output: rg, f5713de20cde511e8ba4900, containerName, namespace, subsID
+func GetContainerInfo(id string) (string, string, string, string, string, error) {
```

Comment:
add comments here:
the format of VolumeId is: `rg#accountName#containerName#uuid#secretNamespace#subsID`